content/docs/news/release_announcements/keptn-0127/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Keptn 0.12.7
+weight: 79
+---
+
+This is a bug fix release for Keptn containing fixes for the shipyard-controller.
```

Comment:
`This is a bug fix release for Keptn addressing a problem with MongoDB client connection handling, an incorrect sequence state, and an issue when deleting a service.`